internal/builtinchecks/yamls/ssh-port.yaml

```diff
@@ -0,0 +1,10 @@
+name: "ssh-port"
+description: "Alert on deployments with containers exposing SSH port 22"
```

Comment:
nit: change to `Alert on deployments exposing port 22, commonly reserved for SSH access.`